deploy/crds/hwameistor.io_localdiskclaims_crd.yaml

```diff
@@ -40,6 +40,11 @@ spec:
           spec:
             description: LocalDiskClaimSpec defines the desired state of LocalDiskClaim
             properties:
+              completed:
```

Comment:
why add "completed" into spec? should it be added into status? If it is added for disk expansion, "completed" may not be a good field name

Reply:
This field is used for scenarios where LDC needs to be expanded. Users need a parameter to trigger the expansion behavior.